index.html

```diff
@@ -6,22 +6,35 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <title>My Portfolio</title>
+  <script src="script.js"></script>
 </head>
 
 <body class="body">
+  <header class="header-container">
+    <a href="" class="my-logo"><img src="./images/My Logo.png" alt="This is my Portfolio Logo"></a>
+    <nav class="navigation">
+      <a href="" class="my-menu"></a>
+      <ul class="desktop-nav">
+        <li class="nav-men">
+          <a href="#myProjects-page" class="nav-link">Portfolio</a>
+        </li>
+        <li class="nav-men">
+          <a href="#myAbouts-page" class="nav-link">About</a>
+        </li>
+        <li class="nav-men">
+          <a href="#myContact-page" class="nav-link">Contact</a>
+        </li>
+      </ul>
+      <div id="mobilenav" class="mobileNavigation">
+        <a href="javascript:void(0)" class="killnav" onclick="menukill()">&times;</a>
+        <a href="#myProjects-page" onclick="closeMenu()">Portfolio</a>
+        <a href="#myAbouts-page" onclick="closeMenu()">About</a>
+        <a href="#myContact-page" onclick="closeMenu()">Contact</a>
+      </div>
```

Comment:
There's too much space between the menu items. Kindly reduce the space between the menu items and try to make it look like the figma design. Also, Please use the required font family as in the Figma design. 👍 

### [YOUR DESIGN]
<img width="396" alt="Screenshot 2023-06-20 at 5 00 39 PM" src="https://github.com/mershark/My-Portfolio/assets/83048208/62e91e85-af56-4c47-8e95-2b2b5a1ccd00">

### [REQUIRED FIGMA DESIGN]
<img width="277" alt="Screenshot 2023-06-20 at 5 02 10 PM" src="https://github.com/mershark/My-Portfolio/assets/83048208/2078cfa8-a7f6-4110-b3f2-8727c5962203">